mypy.ini

```diff
@@ -68,9 +66,6 @@ ignore_errors = true
 # gradually make PlasmaPy consistent with mypy's strict mode, we can
 # remove the disabled error codes file-by-file.
 
```

Comment:
All of the changes in this file made below this line were done automatically, as described in #2424.